probe/endpoint/reporter.go

```diff
@@ -95,26 +96,31 @@ func (r *Reporter) Report() (report.Report, error) {
 
 	hostNodeID := report.MakeHostNodeID(r.hostID)
 	rpt := report.MakeReport()
-	conns, err := procspy.Connections(r.includeProcesses)
-	if err != nil {
-		return rpt, err
-	}
 
-	for conn := conns.Next(); conn != nil; conn = conns.Next() {
-		var (
-			localPort  = conn.LocalPort
-			remotePort = conn.RemotePort
-			localAddr  = conn.LocalAddress.String()
-			remoteAddr = conn.RemoteAddress.String()
-		)
-		extraNodeInfo := report.MakeNode()
-		if conn.Proc.PID > 0 {
-			extraNodeInfo = extraNodeInfo.WithMetadata(report.Metadata{
-				process.PID:       strconv.FormatUint(uint64(conn.Proc.PID), 10),
-				report.HostNodeID: hostNodeID,
-			})
+	{
+		conns, err := procspy.Connections(r.includeProcesses)
+		if err != nil {
+			return rpt, err
+		}
+		commonNodeInfo := report.MakeNode().WithMetadata(report.Metadata{
+			Procspied: "true",
+		})
```

Comment:
Remote addresses appear in /proc/net/tcp, so those two things are the same in my view.